### PR TITLE
Update links to split files

### PR DESCRIPTION
**Pull request by ChrisChinchilla**
A lot done already in other PRs, but added a couple to the tl:dr too

### CODE REVIEW

**Comment by soc1c** [comment]
FFS! 7 minutes is not enough to review!

File: README.md [hunk 1]
```diff
@@ -3,61 +3,72 @@
...
-- Google Maps: <https://goo.gl/maps/ZCEGLYLUSqVYJdMv6>
-- See also: <https://ethberlinzwei.com/>
+The Hackathon takes place between the 23rd and 25th of August at **Factory Görlitzer Park**.
 
+-   Venue: <https://factoryberlin.com/about/>
+-   Street address: Lohmühlenstraße 65, 12435 Berlin, Germany
+-   Google Maps: <https://goo.gl/maps/ZCEGLYLUSqVYJdMv6>
+-   See also: <https://ethberlinzwei.com/>
 
+It's proudly a part of the ETHGlobal community.
```
Comment (soc1c) [inline README.md:40]: We are not?

File: README.md [hunk 2]
```diff
@@ -3,61 +3,72 @@
 This repository of knowledge guides you through getting started with ETHBerlin, Ethereum, blockchain, and decentralized application development.
 
 ## TL;DR!
+
 -   Find a team on Github: <https://github.com/ethberlinzwei/Find-A-Team>
 -   Connect with hackers on Riot: [#ethberlinhackers:matrix.org](https://riot.im/app/#/room/#ethberlinhackers:matrix.org)
 -   Contribute to the knowledge base on Riot: [#ethberlinzwei-kb:chat.weho.st](https://riot.im/app/#/room/#ethberlinzwei-kb:chat.weho.st)
+-   [Find details about bounties and prizes](bounties.md)
+-   [Find out logistics and the schedule of the event](the-event.md)
```
Comment (soc1c) [inline README.md:11]: At some point the TL;DR will become _too long_...

File: README.md (hunk 1 above)
Comment (ligi) [reply README.md:40]: also my information
